.jenkins/Jenkinsfile

```diff
@@ -16,524 +16,486 @@
           }
         }
-        stage('burn') {
+        stage('centos7') {
           steps {
-            script {
-                def attempt = 1
-                while (attempt <=2) {
-                  if (attempt > 1) {
-                    sleep(60 * attempt)
-                  }
-                  attempt = attempt + 1
-                  burn = build job: "${env.JOB_NAME}-test-burn", propagate: false
-                  if (burn.result != 'FAILURE') break
-              }
-              if (burn.result != 'SUCCESS') unstable('burn test failures')
-              if (burn.result == 'FAILURE') currentBuild.result='FAILURE'
-            }
-          }
-          post {
-            always {
-                warnError('missing test xml files') {
-                    script {
-                        copyTestResults('test-burn', burn.getNumber())
-                    }
-                }
-            }
+            buildJob(".build/docker/build-redhat.sh noboolean", "*.rpm")
           }
         }
-        stage('cdc') {
-          steps {
-            script {
-                def attempt = 1
-                while (attempt <=2) {
-                  if (attempt > 1) {
-                    sleep(60 * attempt)
-                  }
-                  attempt = attempt + 1
-                  cdc = build job: "${env.JOB_NAME}-test-cdc", propagate: false
-                  if (cdc.result != 'FAILURE') break
-              }
-              if (cdc.result != 'SUCCESS') unstable('cdc failures')
-              if (cdc.result == 'FAILURE') currentBuild.result='FAILURE'
-            }
-          }
-          post {
-            always {
-                warnError('missing test xml files') {
-                    script {
-                        copyTestResults('test-cdc', cdc.getNumber())
-                    }
-                }
-            }
+       stage('stress') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "stress-test", 1)
+         }
+       }
+       stage('fqltool') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "fqltool-test", 1)
+         }
+       }
+       stage('units') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "test", 8)
+         }
+       }
+       stage('long units') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "long-test", 8)
+         }
+       }
+       stage('burn') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "test-burn", 8)
+         }
+       }
+       stage('cdc') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "test-cdc", 8)
+         }
+       }
+       stage('compression') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "test-compression", 8)
+         }
+       }
+       stage('cqlsh') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "cqlsh-test", 1)
+         }
+       }
+     }
+    }
+    // stage('Distributed Test') {
```

Comment:
Why all commented out? Just interim local testing or something, or is this not yet working?

Reply:
still testing. testing scripts still wip.